src/Symfony/Component/Form/NativeRequestHandler.php

```diff
@@ -230,7 +230,7 @@ private static function fixPhpFilesArray($data)
      *
      * @param mixed $data The file upload data
      *
-     * @return array|null Returns the stripped upload data
+     * @return mixed Returns the stripped upload data
```

Comment:
`$data` is returned as-is if it's not an array. Therefore, if we accept virtually anything as `$data`, we cannot make assumptions on the return type.

Reply:
`stripEmptyFiles()` is called with the result of `fixPhpFilesArray()` which would have to be adapted in the same way, right?

Reply:
`fixPhpFilesArray` implements the same pattern, so yes, I'm going to adjust that method as well.